uber/custom_tags.py

```diff
@@ -445,12 +445,11 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
-        skiprest = ''
         for day, price in BADGE_PRICES['single_day'].items():
             if day == datetime.strftime(ESCHATON, "%A"):
                 prices += 'and ${} for {}'.format(price, day)
-                skiprest = 'YES'
-            elif skiprest == '':
+                break
+            else
```

Comment:
I suspect that this whole tag is simply not being used, because it's referring to ``ESCHATON`` instead of ``c.ESCHATON`` and ``BADGE_PRICES`` instead of ``c.BADGE_PRICES``, etc.